src/main/java/com/epam/lowcost/DAO/PlaneDAOImpl.java

```diff
@@ -0,0 +1,104 @@
+package com.epam.lowcost.DAO;
+
+import com.epam.lowcost.model.Plane;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaneDAOImpl implements PlaneDAO {
+
+    private DataSource dataSource;
```

Comment:
There are some changes in develop branch with datasource. Merge it into your branch and resolve conflicts

Reply:
Merged and resolved